From 0daf4ae637677443c1b3a2a9d5e48cabe1cb1418 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Tue, 18 Apr 2023 15:53:11 +0200
Subject: [PATCH] tests: unset CFLAGS

CFLAGS may be set in the running environment (eg in abuild check)
---
 tests/functions_test | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/functions_test b/tests/functions_test
index b7928de..fd6e594 100755
--- a/tests/functions_test
+++ b/tests/functions_test
@@ -60,6 +60,8 @@ funcs_check_defconfig_body() {
 	echo "CFLAGS=syscflags" > etc.conf
 	echo "CFLAGS=usercflags" > user.conf
 
+	unset CFLAGS
+
 	# test /usr/share/abuild/default.conf
 	ABUILD_DEFCONF="$PWD"/def.conf ABUILD_CONF=/dev/null ABUILD_USERCONF=/dev/null \
 		atf_check -o match:"defcflags" \
-- 
2.40.0

